mllib/src/test/scala/org/apache/spark/ml/tree/impl/RandomForestSuite.scala

```diff
@@ -362,10 +365,10 @@ class RandomForestSuite extends SparkFunSuite with MLlibTestSparkContext {
     assert(topNode.isLeaf === false)
     assert(topNode.stats === null)
 
-    val nodesForGroup = Map((0, Array(topNode)))
-    val treeToNodeToIndexInfo = Map((0, Map(
-      (topNode.id, new RandomForest.NodeIndexInfo(0, None))
-    )))
+    val nodesForGroup = Map(0 -> Array(topNode))
```

Comment:
These are fine, but I slightly prefer leaving stuff like this out. These aren't strictly style violations, and it distracts reviewers from the actual changes.

Reply:
That was at my request, because we made that change in some new code, and then made a little sense to improve this also for consistency.

Reply:
Two tests previously moved here have now been moved back, there is still
_"Use soft prediction for binary classification with ordered categorical features"_ to which I have applied @srowen 's comment, so the consistency argument still holds (even if weakened a bit).